.circleci/config.yml

```diff
@@ -138,7 +138,31 @@ jobs:
       - run:
           name: Unit Tests
           shell: powershell.exe
-          command: go test -race -coverprofile coverage.txt -coverpkg ./... ./...
+          command: >
```

Comment:
I'm certain there is a better way to do this that doesn't involve listing each module manually, but I figure we can handle that later.